tests/install/install_ps1_test.py

```
@@ -111,6 +123,17 @@ def main() -> None:
         raise RuntimeError("install_ps1_test.py is a Windows-only test")
     powershell = sys.argv[1]
     binary = Path(sys.argv[2]).resolve()
+    # The fixture packages this binary as the "cpu" release. install.ps1 runs
+    # `doctor` as a health check, which fails when the binary carries a GPU
+    # backend but the host exposes no GPU. That is correct installer behavior
+    # and unrelated to what this test covers, so skip rather than fail.
+    doctor = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)